.circleci/config.yml

```diff
@@ -7,6 +7,8 @@ jobs:
         type: string
     docker:
       - image: cimg/node:18.17
+        environment:
+          PISA_URL: 'some-fake-pisa-url'
```

Comment:
* [ ] Even with the commit message it's unclear what this fake setting is about. I personally never looked at any filesize in the CI environment. Maybe we don't need this or there should be a helpful comment in the config.yml?